web/satellite/src/components/AccessTableComponent.vue

```diff
@@ -58,7 +58,7 @@
                         <v-list class="pa-1">
                             <v-list-item class="text-error" density="comfortable" link @click="() => onDeleteClick(item)">
                                 <template #prepend>
-                                    <icon-trash />
+                                    <component :is="Trash2" :size="18" />
```

Comment:
`<v-icon :size="18" :icon=Trash2 />` instead?

Reply:
Hey Wilfred, using the v-icon was adding some weird margins in some places for the icons, and this worked as it should.

Reply:
Noted